business-central-tests/business-central-tests-rest/src/test/java/org/kie/wb/test/rest/functional/ProjectIntegrationTest.java

```diff
@@ -95,6 +97,7 @@ public void testCreateProjectWithVersion() {
         createProject("projectWithoutVersion", null, GROUP_ID, VERSION);
     }
 
+    @Ignore
```

Comment:
please add here a reason and jira URL you created for fixing the tests in future